tests/assets/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="utf-8" />
+    <meta http-equiv="x-ua-compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <title>ostrich example</title>
+
+    <style>
```

Comment:
We probably dont need any styling for the test, right?

Reply:
The test sites need just the bare minimum.